cl/corpus_importer/management/commands/harvard_merge.py

```diff
@@ -941,6 +973,11 @@ def add_arguments(self, parser) -> None:
             action="store_true",
             help="A flag to choose to merge only fastcase opinions",
         )
+        parser.add_argument(
+            "--merge-judges",
+            action="store_true",
+            help="Set flag to merge judges",
+        )
```

Comment:
I think a better name would be smart.  

Im guilty of this more than most but we want this flag to be used to skip merging of judge data if enabled.  right?  

so something like `skip_judge_merger` would be more understandable-  or `merge_without_judge_match` to illustrate that we are enabling a merge even if it doesnt match.  ? this would change the store true to store false I think but I think it makes sense

Reply:
maybe `enable-judge-merger`? By default, it is disabled unless you pass the option to the command

Reply:
I think my issue with this - is that the default is always to merge judges-  Its more of a flag to enable the completion of the merger without the judge if the judge doesnt match (presumably for typos).   soI think  the flag should reflect that.